Stonks/Forms/FormFinances.cs

```diff
@@ -135,12 +135,9 @@ private void buttonSaveFinances_Click(object sender, EventArgs e)
                 textBoxOther.Text = Convert.ToString(otherExpense.Value);
             }
 
-            //Calculate Savings
-            //FIX LATER
-            //Program.financialPlan.Savings = Program.financialPlan.Income - (housingExpense.Value + groceries.Value + transportExpense.Value + entertainmentExpense.Value + healthExpense.Value + utilitiesExpense.Value + otherExpense.Value + shoppingExpense.Value);
         }
 
-        public void saveExpenseValues()
+        public void saveExpenseValues() //Answer the question with team, do we want planned values to be 0 from the start or set to the values of the actual expenses of the user
```

Comment:
I am for setting to the values of the actual expenses of the user

Reply:
I agree. It's logical that both values would be the same in the beginning.